geometry_msgs/Pose2D home1
geometry_msgs/Pose2D home2
geometry_msgs/Pose2D away1
geometry_msgs/Pose2D away2
geometry_msgs/Pose2D ball 